pkg/pull/pull.go

```diff
@@ -332,13 +333,13 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		renderDir = filepath.Join(pullOptions.RootDir, u.Name)
 	}
 
-	newHelmCharts, err := kotsutil.LoadHelmChartsFromPath(renderDir)
```

Comment:
shouldn't we enforce this for v1beta2 as well? e.g.: can't migration from old replicated Helm to v1beta1 _or_ v1beta2 native Helm

Reply:
yeah I'll add that in for v1beta2 charts as well